canvas_modules/common-canvas/locales/command-actions/locales/eo.json

```diff
@@ -16,14 +16,14 @@
 	"action.createNodeOnLink": "[Esperanto~Create {node_label} node and insert into link~~~~~~~eo]",
 	"action.createSuperNode": "[Esperanto~Create {node_label} supernode~~~~~~~eo]",
 	"action.deconstructSuperNode": "[Esperanto~Deconstruct {node_label} supernode~~~~~~~eo]",
-	"action.deleteComments":"[Esperanto~Delete {comments_count} comments~~~eo]",
-	"action.deleteNodes":"[Esperanto~Delete {nodes_count} nodes~~~eo]",
+	"action.deleteComments":"[Esperanto~Delete {comments_count, plural, one {# comment} other {# comments} }~~~eo]",
+	"action.deleteNodes":"[Esperanto~Delete nodes_count, plural, one {# node} other {# nodes} }~~~eo]",
 	"action.deleteLinks":"[Esperanto~Delete {links_count} links~~~eo]",
-	"action.deleteCommentsLinks":"[Esperanto~Delete {comments_count} comments and {links_count} links~~~~eo]",
-	"action.deleteNodesLinks":"[Esperanto~Delete {nodes_count} nodes and {links_count} links~~~~eo]",
+	"action.deleteCommentsLinks":"[Esperanto~Delete {comments_count, plural, one {# comment} other {# comments} } and {links_count, plural, one {# link} other {# links} }~~~~eo]",
+	"action.deleteNodesLinks":"[Esperanto~Delete {nodes_count, plural, one {# node} other {# nodes} } and {links_count, plural, one {# link} other {# links} }~~~~eo]",
 	"action.deleteLink": "[Esperanto~Delete link~~~~~~~eo]",
-	"action.deleteNodesCommentsLinks": "[Esperanto~Delete {nodes_count} nodes, {comments_count} comments and {links_count} links~~~~~~~eo]",
-	"action.deleteNodesComments": "[Esperanto~Delete {nodes_count} nodes and {comments_count} comments~~~~~~~eo]",
+	"action.deleteNodesCommentsLinks": "[Esperanto~Delete {nodes_count, plural, one {# node} other {# nodes} }, {comments_count, plural, one {# comment} other {# comments} } and {links_count, plural, one {# link} other {# links} }~~~~~~~eo]",
+	"action.deleteNodesComments": "[Esperanto~Delete {nodes_count, plural, one {# node} other {# nodes} } and {comments_count, plural, one {# comment} other {# comments} }~~~~~~~eo]",
 	"action.disconnectObjects": "[Esperanto~Disconnect objects~~~~~~~eo]",
```

Comment:
Something is wrong with this file. There should be the same number of changed lines in this file as in the English version of the file. Please check carefully to make sure the changes are the same in both files.